{"name": "test_invalid_login", "status": "passed", "description": "Test login with incorrect email and password.", "attachments": [{"name": "log", "source": "7cf0ded0-bfce-4342-8c23-331f817a667c-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "063b5d01-1629-40f5-9fbf-9154a46c2a1c-attachment.txt", "type": "text/plain"}], "start": 1737939460531, "stop": 1737939462009, "uuid": "ca1bca86-6fa2-4e89-a503-fdb5ea2bb546", "historyId": "27ef76719e6ec0ddbdf744d3722f4c79", "testCaseId": "27ef76719e6ec0ddbdf744d3722f4c79", "fullName": "tests.api_tests.users.test_users_api_negative#test_invalid_login", "labels": [{"name": "parentSuite", "value": "tests.api_tests.users"}, {"name": "suite", "value": "test_users_api_negative"}, {"name": "host", "value": "Dima"}, {"name": "thread", "value": "20172-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api_tests.users.test_users_api_negative"}]}